core/app/components/pubs/PubEditor/helpers.ts

```diff
@@ -0,0 +1,27 @@
+import type { FormElementsId } from "db/public";
```

Comment:
I removed this in https://github.com/pubpub/platform/pull/820 but turns out I still needed it, for pub values that don't have a corresponding form! no changes were made here